### DISCUSSION TITLE
Would you use the discussions on GitHub?

### DISCUSSION THREAD

**Discussion by grymmjack** [QB64-Phoenix-Edition/QB64pe#375] [Polls]
# Testing to see if people would use discussions on GitHub?

Why would you want to do this?

1. Markdown support
2. Paste image support
3. So much easier to use than MyBB
4. Costs $0

Built in issue tracker support by #376

@a740g at mentions built in

```vb
PRINT "CODE HIGHLIGHTING BUILT IN"
```

Attachment support...
[README.md](https://github.com/QB64-Phoenix-Edition/QB64pe/files/12706721/README.md)

**Comment by SteveMcNeill** [comment]
Honestly, I'd prefer not to.  We already have the forums and the Discord.  All having Discussions here would do is fracture the conversations over to yet another place that people would have to get used to visiting and responding, and there's a large portion of the user base who can't figure out how to even type www.github.com into their browser, for whatever reason.  

That said, I'll also say I'm too much of a busybody not to respond *wherever* someone comments and I notice it.  If someone starts a topic/discussion here, then I'm -- of course -- going to have to put my two cents in!  My only objection here is that I think trying to move discussions to take place on Github on a regular basis, obsoletes or depletes the purpose of having them elsewhere.  Public conversations tend to take place on the forums, where the most number of people visit and interact regularly.  Private conversations between developers tend to take place in our Discord, where we toss ideas back and forth and share insights and such with each other for quick responses and interactions.

IF Github was to be used as a place for general discussions, what exactly would be its role?  To replace or redirect traffic from the forums to here?  To let developers have conversations here instead of on Discord?  Something else, which I haven't thought of?

Personally, I don't feel like I need yet another place to check  regularly to converse with folks...  And yet, with that aside, as long as I notice someone has posted here, I'm going to end up reading it and sharing my opinion/insight on whatever the topic in question is.  

So I guess I'd use it, but being an old fart, I reserve the right to grumble moodily while doing so.  LOL!  ;)

**Reply by a740g** [reply]
I mean GitHub is where most of our workflow lives. Everything integrates nicely with each other. Anyone who is contributing to QB64-PE can use these features. But with that said, we need the forum too for folks who don't like or want to use GitHub and / or Discord.

**Reply by RhoSigma-QB64** [reply]
Voted FALSE for the very same reasons Steve mentioned.
Having inline MD, IMG, CODE support, well we also have it in the other two places too.
Also MD is not the premier solution, Matt already tried that for the wiki last year, but we had ended with a plain wiki like Cory has in his "official" repo.

**Comment by grymmjack** [comment]
@SteveMcNeill answering a few of your Q's:

The idea would be to stop having a forum, and use this instead.  I guess it sounds like no one wants that, so all good and no problem. 

I would much rather use this discussions board instead of our MyBB forums. I realize I am a minority opinion here so I will just stop trying to evolve things and leave things be.

I don't buy the idea that "They don't know how to use GitHub" argument. These people are smart enough to figure out how to compile programs and write them, they could figure out how to use GitHub. bplus is now on Discord; did you know that? So this idea of people not adapting and putting their head in the sand, while it has merit, it also is possible to just cajole them into the future. I guess it doesn't really matter, but I'm really frustrated as a end-user of the forums we have. I am spoiled by the modern ways of things every where else.

Ignore this then :)

All good, continue on doing the same things..
